static/panes/cfg-view.interfaces.ts

```diff
@@ -22,17 +22,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import {PaneState} from './pane.interfaces';
-import * as vis from 'vis-network';
-
-export interface CfgOptions {
-    physics?: boolean;
-    navigation?: boolean;
-}
-
-export interface CfgState extends PaneState {
-    selectedFn?: string;
-    pos: vis.Position;
+export interface CfgState {
+    selectedFunction: string;
```

Comment:
This change will make it incomptabile with old cfg links. No way around it?

Reply:
Thanks for the reminder, I think it's doable to provide backward compatibility here just a few extra cases to handle in the code. I'll flesh this stuff out more later.